crates/torii/indexer/src/processors/upgrade_model.rs

```diff
@@ -30,6 +32,17 @@ where
         true
     }
 
+    fn task_priority(&self) -> usize {
+        0
+    }
+
+    fn task_identifier(&self, event: &Event) -> u64 {
+        let mut hasher = DefaultHasher::new();
+        // model selector
+        event.keys[1].hash(&mut hasher);
```

Comment:
In some other events we don't have the comment.
Wondering if it would be better unwrapping the cainome deserialization and using the fields which would make the hasher more explicit.

However, you may have detected performance issues with cainome being perhaps too heavy at this point of the code, which may explain why using direct access?
In this case, having comments could help, even if it will not help if some code is not updated on event changes (which shouldn't occur that much though).

Reply:
Cainome deserializes all keys and data and in there I didn't think it would make sense using it. I only use cainome when dealing with Bytearrays or complex data types. For now I think it is fine. In the future we might change the event processor structure to have a cainome preprocessor to parse the data and pass it to the process/ task identifier functions